dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/cli/CLIUtil.java

```diff
@@ -214,20 +208,14 @@ private static void checkErrorResult(String[] command, ProcessResult result) thr
 		try {
 			if (result.getOutput() != null && !result.getOutput().isEmpty()) {
 				JSONObject obj = new JSONObject(result.getOutput());
-				if (obj.has(ERROR_KEY)) {
```

Comment:
Shouldn't this handle the case that `obj.has(ERROR_KEY)` is true but `obj.has(ERROR_DESCRIPTION_KEY)` is false?  If it's ok to ignore that case, maybe there should be a comment explaining why it's ok.

Reply:
I added a comment and merged the 2 methods so it is easier to see what it is doing.  Until the output from cwctl is all normalized I did not want to depend solely on the existence of an "error" key which seems too generic.